pyccel/ast/datatypes.py

```diff
@@ -81,7 +81,19 @@
     "logical" : {
         4  : "C_BOOL"}
 }
-
+iso_c_binding_shortcut_mapping = {
+    'C_INT8_T'              : 'i8',
+    'C_INT16_T'             : 'i16',
+    'C_INT32_T'             : 'i32',
+    'C_INT64_T'             : 'i64',
```

Comment:
Missing `C_INT128_T` (as mentioned above this also needs correcting in `iso_c_binding`)